### PR TITLE
reboot awesome ipfs

### PR DESCRIPTION
**Pull request by 2color**
## What kind of PR is this?

- Reboot the list while reducing the maintenance overhead
- Kill zombies (dead links and ancient and unmaintained projects)
- Use markdown only with no static site
- Update the workflows to only run against `readme.md`

### Notes
- datasets and articles have been removed to keep this focused on projects/services
- Many of the projects here are long inactive and probably not working and take the thunder away from actively maintained awesome stuff — those have been removed. The repo still has the full history for those interested. 
- The workflow to remove broken links has been removed as it won't work with the simple README.md format. **This is not a problem** because we will still get the results from the awesome_bot in the comments of PRs via danger ([example](https://github.com/ipfs/awesome-ipfs/pull/477#issuecomment-1715108874)).

### CODE REVIEW

**Comment by meandavejustice** [comment]
After looking over the list I think there should be a section for stale/deprecated projects that are significant enough to keep around as a demo. For example ipfs-search & ipfs-chat

**Comment by meandavejustice** [comment]
Link to line that reports for README edits on pull requests https://github.com/ipfs/awesome-ipfs/blob/master/dangerfile.js#L10

File: POLICY.md [hunk 1]
```diff
@@ -6,10 +6,8 @@ Entries are accepted if they meet the following criteria:
 
 * Entries must respect the [IPFS Community Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md)
 * Entries must use *IPFS* (libp2p, ipld, multiformats is not enough)
+* Entries in the **apps** & **tools** categories must be **open-source** and contain a link to their source code.
 * Entries must be immediately usable (eg. nothing that is paywall-only, even a free trial would be ok)
 * Entries must have a clear license and/or privacy policy where applicable
 
-This policy is evaluated by the site admins upon submission, per the process defined in the [contribution guide](CONTRIBUTING.md).
-
-This policy was developed [here](https://github.com/ipfs/awesome-ipfs/issues/150) - please add questions, comments and suggestions there.
```
Comment (SgtPooki) [inline POLICY.md:14]: Should we link to this PR in that issue and mention that there was an update to policy?

File: README.md [hunk 2]
```diff
@@ -1,233 +1,71 @@
 # Awesome IPFS [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
-> Useful resources for using [IPFS](https://ipfs.tech) and building things on top of it
+This is a community list is for awesome projects, apps, tools, and services related to IPFS.
```
Comment (SgtPooki) [inline README.md:3]:
```suggestion
This is a community list for awesome projects, apps, tools, and services related to IPFS.
```

File: README.md [hunk 3]
```diff
@@ -1,233 +1,71 @@
...
-- 2017-05-05: [Decentralized indexes for public genomic data](https://github.com/luizirber/2017-recomb) 
-- 2017-03-31: [Using IPFS for IoT Communications](https://medium.com/@chrismatthieu/using-ipfs-for-iot-communications-b49c2139783a) 
-- 2016-09-12: [IPFS: The Internet Democratised](https://medium.com/@tonywillenberg/web-3-0-a-truly-democratised-internet-f4b06cb4077b) 
-- 2016-05-20: [Changelog Podcast](https://changelog.com/podcast/204/) 
-- 2015-09-18: [The InterPlanetary File System Wants to Create a Permanent Web](http://motherboard.vice.com/read/the-interplanetary-file-system-wants-to-create-a-permanent-web) 
-- 2015-09-13: [Introduction to IPFS](http://whatdoesthequantsay.com/2015/09/13/ipfs-introduction-by-example) 
-- 2015-09-08: [HTTP is obsolete. It's time for the distributed, permanent web](https://ipfs.io/ipfs/QmNhFJjGcMPqpuYfxL62VVB9528NXqDNMFXiqN5bgFYiZ1/its-time-for-the-permanent-web.html) 
-- [Downloading nodejs versions with nvm/n over IPFS](https://ipfs.io/ipfs/QmTkzDwWqPbnAh5YiV5VwcTLnGdwSNsNTn2aDxdXBFca7D/example#/ipfs/QmUx363UFtgiQqkHHsPK3TSDmwoALDo2hrbMWbcxjH2vFc) 
-
-## Datasets
```
Comment (SgtPooki) [inline README.md:153]: Might be useful to mention a place where these items can go in the future. IIRC there was a filecoin/ipfs datasets website somewhere

File: README.md [hunk 4]
```diff
@@ -1,233 +1,71 @@
...
-- [Blockwatch](https://ipfs.io/ipfs/QmdikpwcyeBuGaVzWzSzPuqvBfTGD8jPAVydcCjYHsBUxo/index.html) - Monitor the current block number of the Ethereum blockchain, and set alerts (stored locally using PouchDB in your browser) for when certain block heights are passed. [Source](https://github.com/MidnightLightning/ethereum-blockwatch)
-- [Bubble IPFS From URL Plugin](https://bubble.io/plugin/ipfs-add-from-uri-1616424944051x638386852081172500) - Bubble (no code platform) plugin for IPFS From URL integration
-- [cachewarmer](https://github.com/BrendanBenshoof/cachewarmer) - Donate ipfs gateways to cache other people's content
 - [create-ipfs-app](https://github.com/alexbakers/create-ipfs-app) - Set up a decentralized web3 app by running one command.
-- [galacteek](https://galacteek.github.io) - A multi-platform Qt5-based browser for the distributed web [Source](https://github.com/pinnaculum/galacteek)
+- [galacteek](https://github.com/pinnaculum/galacteek) - A multi-platform Qt5-based browser for the distributed web.
```
Comment (SgtPooki) [inline README.md:44]: Is this an App instead of a tool?

**Review by SgtPooki — COMMENTED** [review]
LGTM, no approval because CI is still failing.

**Comment by 2color** [comment]
Thanks for the review @SgtPooki. CI is failing because this PR breaks some of the old workflow/danger config. After this PR is merged, it should be sorted.

File: POLICY.md (hunk 1 above)
Comment (2color) [reply POLICY.md:14]: Sure! Just did it: https://github.com/ipfs/awesome-ipfs/issues/150#issuecomment-1729602722

File: README.md (hunk 4 above)
Comment (2color) [reply README.md:44]: I'll move it to apps!

**Pull request merged**